scram-tools.file/tools/EMTF_NN/emtf_nn.xml

```diff
@@ -0,0 +1,6 @@
+<tool name="emtf_nn" version="@TOOL_VERSION@" revision="1">
+  <client>
+    <environment name="EMTF_NN_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR"       default="$EMTF_NN_BASE/lib64"/>
```

Comment:
```suggestion
<tool name="@TOOL@" version="@TOOL_VERSION@" revision="1">
  <client>
    <environment name="@TOOL_BASE@" default="@TOOL_ROOT@"/>
    <environment name="LIBDIR"       default="@TOOL_BASE@/lib64"/>
```

@pallabidas , can you also please apply this change? build/packaging system will replace `@VAR@` with the correct values.